### ISSUE TITLE
no module

### ISSUE THREAD

**Issue by joveice** [AHAAAAAAA/PokemonGo-Map#1333]
Running linux Ubuntu Server 14.04.3 LTS.
Getting this when trying to start.

Traceback (most recent call last):
  File "runserver.py", line 13, in <module>
    from pogom.app import Pogom
  File "/home/***/***/pogom/app.py", line 8, in <module>
    from s2sphere import *
ImportError: No module named s2sphere

**Comment by Bakies** [comment]
install the dependencies and see if an error occurs

**Comment by Faustvii** [comment]
If unsure how to install dependencies, see [Wiki](https://github.com/AHAAAAAAA/PokemonGo-Map/wiki) for a guide.

**Comment by joveice** [comment]
Have installed with pip install -r requirements.txt

**Comment by joveice** [comment]
So ran pip install s2sphere, then it asked for peewee when starting, so did the same for that, then it asked for a google something. That wasent found when using pip.

**Comment by joveice** [comment]
Im now at No module named protobuf_to_dict. What am I missing?

**Comment by snagaduck** [comment]
If you were setting up a new computer, I had to do my steps a little different than the guide to make it work on Ubuntu 14.04...

Here is what I had to do... I kept getting errors on the pip install. Must have been a missing permission, so I added common dependencies to my server. And everything seemed to go right through.

apt-get update
apt-get upgrade
apt-get install emacs-nox
apt-get install python python-setuptools
apt-get install build-essential autoconf libtool pkg-config python-opengl python-imaging python-pyrex python-pyside.qtopengl idle-python2.7 qt4-dev-tools qt4-designer libqtgui4 libqtcore4 libqt4-xml libqt4-test libqt4-script libqt4-network libqt4-dbus python-qt4 python-qt4-gl libgle3 python-dev
easy_install pip
mkdir pokemon
cd pokemon
wget https://github.com/AHAAAAAAA/PokemonGo-Map/archive/develop.zip
unzip develop.zip
cd PokemonGo-Map-develop/
pip install -r requirements.txt
